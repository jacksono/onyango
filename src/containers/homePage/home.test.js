import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
import Home from './home';
/* eslint no-undef:0 */
test('Home page renders correctly', () => {
  const tree = renderer
    .create(
      <MuiThemeProvider>
        <Router>
          <Home />
        </Router>
      </MuiThemeProvider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
