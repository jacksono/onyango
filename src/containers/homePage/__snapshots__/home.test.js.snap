// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page renders correctly 1`] = `
<div
  className="page"
>
  <h1>
    Note Titles By Others
  </h1>
  <hr
    style={
      Object {
        "backgroundColor": "#e0e0e0",
        "border": "none",
        "height": 1,
        "margin": 0,
        "marginBottom": "10px",
        "marginLeft": 0,
        "marginTop": -1,
        "muiPrepared": true,
      }
    }
  />
  <span>
    There are no notes yet.
    <a
      href="/new"
      onClick={[Function]}
    >
       Click here 
    </a>
    to add the first one.
  </span>
  <div />
</div>
`;
